postgres-appliance/scripts/configure_spilo.py

```diff
@@ -855,9 +855,9 @@ def write_crontab(placeholders, overwrite):
         logging.info('Skipping creation of renice cron job due to lack of SYS_NICE capability')
 
     if placeholders.get('SSL_TEST_RELOAD'):
-        env = ' '.join('{0}="{1}"'.format(n, placeholders[n]) for n in ('SSL_CA_FILE', 'SSL_CRL_FILE',
+        env = ' '.join('{0}="{1}"'.format(n, placeholders[n]) for n in ('PGDATA', 'SSL_CA_FILE', 'SSL_CRL_FILE',
```

Comment:
Why do we need PGDATA here?
It doesn't seem to be referenced from the test_reload_ssl.sh

Reply:
PGDATA is not available in the scope of the script if it is not explicitly set by the cronjob. I am not sure why/how this worked before, but for me the reload script fails without this.

```
Reloading due to detected changes
pg_ctl: no database directory specified and environment variable PGDATA unset
Try "pg_ctl --help" for more information.
```